packages/expandable/src/ux-expandable.ts

```diff
@@ -61,22 +47,17 @@ export class UxExpandable implements UxComponent {
 
   updateContainerHeight() {
     if (this.openBoolean) {
-      // after transition set body height to auto so that expandable children are visible
-      this.contentContainer.addEventListener("transitionend", this);
-      this.contentContainer.style.height = this.content.clientHeight + "px";
+      this.contentContainer.style.height = this.content.clientHeight + 'px';
     } else {
-      // the following line is needed because height has been restored to auto"
-      this.contentContainer.style.height = this.content.clientHeight + "px";
-      this.taskQueue.queueTask(() => {
-        this.contentContainer.style.overflow = "hidden";
-        this.contentContainer.style.height = "0";
-      });
+      this.contentContainer.style.height = '0';
     }
   }
 
   toggle() {
-    if (!this.openBoolean && normalizeBooleanAttribute('accordion', this.accordion)) {
-      const otherAccordions = Array.from(this.element!.parentElement!.querySelectorAll('ux-expandable[accordion].ux-expandable--open'));
+    if (!this.openBoolean && this.accordion !== undefined) {
+      const otherAccordions = this.accordion === ''
+        ? Array.from(this.element!.parentElement!.querySelectorAll('ux-expandable[accordion].ux-expandable--open'))
```

Comment:
I just realized that using `.querySelectorAll` will select all expandable. Even the ones nested inside this, is this the intention?

Reply:
Not even sure if nested expandables would work. So no, there was no such intent

Reply:
Children expandables work fine. To prevent children from collapsing one has to set `accordion` attribute to some unique value on a group of collapsibles.